src/pages/AdminManageUsers.js

```diff
@@ -29,7 +29,7 @@ class AdminManageUsers extends Component {
   }
 
   async getUsers() {
-    const users = await User.list();
+    const users = await User.list("Admin");
```

Comment:
Have a look at `utils.js`, `USER_ROLES` where we have constants for these values.
```suggestion
    const users = await User.list(USER_ROLES.admin);
```